install/kubernetes/cilium/README.md

```diff
@@ -18,7 +18,7 @@ efficient and flexible.
 
 ## Prerequisites
 
-* Kubernetes: `>= 1.16.0-0`
+* Kubernetes: `>= 1.19.0-0`
```

Comment:
Are we finally dropping support for older K8s versions? If yes, I think we'll want to update the page dedicated to K8s compatibility as well.

Reply:
The docs page already says this, this is just updating the helm charts + corresponding docs. I'd like to move it even higher, but this is just making it consistent with the docs. 

https://docs.cilium.io/en/stable/network/kubernetes/compatibility/#k8scompatibility

Reply:
Oh wow it seems like compatibility table actually already is set to 1.26+ since 3 weeks ago 😱 So maybe we actually want 1.26 here.

Reply:
Oh, I missed that. I can bump this in Helm.